src/support/global.c

```diff
@@ -40,6 +40,72 @@ __wt_endian_check(void)
 	return (EINVAL);
 }
 
+/*
+ * __global_calibrate_ticks --
+ *	Calibrate a ratio from rdtsc ticks to nanoseconds.
+ */
+static void
+__global_calibrate_ticks(void)
+{
+#if defined (__i386) || defined (__amd64)
+	struct timespec start, stop;
+	double ratio;
+	uint64_t diff_nsec, diff_tsc, min_nsec, min_tsc;
+	uint64_t tries, tsc_start, tsc_stop;
+	volatile uint64_t i;
+
+	min_nsec = min_tsc = 0;
```

Comment:
This initialization seems like it would go better near the loop in which it's used, setting it to `UINT64_MAX` means you don't have to explicitly test for 0, `WT_MIN` always works (shrug).